src/ImpactX.cpp

```diff
@@ -108,6 +113,34 @@ namespace impactx
         pp_algo.queryAdd("space_charge", space_charge);
         amrex::Print() << " Space Charge effects: " << space_charge << "\n";
 
+        amrex::ParmParse pp_impactx("impactx");
```

Comment:
This should not be in `evolve ()` since that is too late.
It could be a function called at the end of `ImpactX::ImpactX`